spec/features/category_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+feature 'Category page' do
+  before :each do
+    visit categories_path
+  end
+
+  scenario 'has appropriate content' do
+    expect(page).to have_content I18n.t(:catalog)
+    expect(page).to have_current_path categories_path
+
+    expect(page).to have_link I18n.t(:all)
+  end
+
+  scenario 'when click onto a category proceed to Category page' do
+    find('a.filter-link').click
```

Comment:
choose link not only by a selector. Add for test spect option
`find('a.filter-link', text: category.title)`

Reply:
check that we can see only current category books